gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -25,17 +25,22 @@
 import logging
 
 from collections import defaultdict
-from typing import Dict, List, Any  #pylint: disable=unused-import
+from typing import Dict, List, Any  # pylint: disable=unused-import
 
 import vcf
 
 from apache_beam.io.gcp.internal.clients import bigquery
 from gcp_variant_transforms.beam_io import vcfio
+from gcp_variant_transforms.libs import beam_util
 from gcp_variant_transforms.libs import bigquery_util
-from gcp_variant_transforms.libs import vcf_header_parser  #pylint: disable=unused-import
+from gcp_variant_transforms.libs import vcf_header_parser  # pylint: disable=unused-import
 
 
 _FIELD_COUNT_ALTERNATE_ALLELE = 'A'
+# Counter names
+_VARIANT_COUNTER = 'variant_counter'
```

Comment:
Consider creating a class for this to group them together. You can also remove the _COUNTER suffix in that case (we use this pattern in other files like BigQuery constants).